tools/incubator.js

```diff
@@ -15,6 +15,8 @@ const Incubator = new Lang.Class({
         this._basin_manifest_path = basin_manifest_path;
         this._manifest = this._read_json_file(GLib.build_filenamev([this._hatch_dir, 'hatch_manifest.json']));
         this._sets = this._read_json_file(GLib.build_filenamev([this._hatch_dir, 'hatch_sets.json']));
+        if (!('tags' in this._sets))
```

Comment:
You need to ensure that `this._sets` is not `null`, in case the file doesn't exist.